docs/releasehistory.md

```diff
@@ -55,6 +100,90 @@ print(value_roundtrip)
+- NonbondedMethod
+- ParameterHandler.assign_parameters()
+- ParameterHandler.postprocess_system()
+- ParameterHandler.check_partial_bond_orders_from_molecules_duplicates()
+- ParameterHandler.assign_partial_bond_orders_from_molecules()
+
+ -->
+
+### Breaking change: `Topology` molecule representation
+
+`Topology` objects now store complete copies of their constituent `Molecule` objects, rather than using simplified classes specific to `Topology`. This dramatically simplifies the code base and allows the use of the full `Molecule` API on molecules inside topologies.
+
+The following classes have been removed:
+- `TopologyAtom`
+- `TopologyBond`
+- `TopologyMolecule`
+- `TopologyVirtualParticle`
+- `TopologyVirtualSite`
+
+In addition, the `Topology.identical_molecule_groups` property has been added, to facilitate iterating over copies of isomorphic molecules in a `Topology`.
+
+### Breaking change: Removed virtual site handling from topologies
+
+To maintain a clear distinction between a model and the chemistry it represents, virtual site handling has been removed from the Toolkit's `Topology` and `Molecule` classes. Virtual site support remains in the force field side of the toolkit, but creating virtual sites for particular molecules is now the responsibility of OpenFF Interchange. This allows the same `Topology` to be used for force fields that use different virtual sites; for example, a topology representing a solvated protein might be parametrized separately with a 3-point and 4-point water model.
+
+As part of this change, the distinction between `Atom` and `Particle` is deprecated. The `Particle` class will be removed in a future release.
+
+The following classes have been removed:
+- `BondChargeVirtualSite`
+- `DivalentLonePairVirtualSite`
+- `MonovalentLonePairVirtualSite`
+- `TrivalentLonePairVirtualSite`
+- `VirtualParticle`
+- `VirtualSite`
+
+The following methods and properties have been removed:
+- `Atom.add_virtual_site()`
+- `Atom.virtual_sites`
+- `FrozenMolecule.compute_virtual_site_positions_from_conformer()`
+- `FrozenMolecule.compute_virtual_site_positions_from_atom_positions()`
+- `FrozenMolecule.n_virtual_sites`
+- `FrozenMolecule.n_virtual_particles`
+- `FrozenMolecule.virtual_sites()`
+- `Molecule.add_bond_charge_virtual_site()`
+- `Molecule.add_monovalent_lone_pair_virtual_site()`
+- `Molecule.add_divalent_lone_pair_virtual_site()`
+- `Molecule.add_trivalent_lone_pair_virtual_site()`
+- `Molecule.add_bond_charge_virtual_site()`
+- `Topology.n_topology_virtual_sites`
+- `Topology.topology_virtual_sites`
+- `Topology.virtual_site()`
+- `Topology.add_particle()`
+
+### Breaking change: Removed `Topology.charge_model` and `Topology.fractional_bond_order_model`
+
+To maintain a clear distinction between a model and the chemistry it represents, the `Topology.charge_model` and `Topology.fractional_bond_order_model` properties have been removed. Charge models and FBOs are now the responsibility of the ForceField.
```

Comment:
I wonder if it would be useful to add (a smithed version of) "these properties weren't even used anywhere lol"; it might be worth emphasizing that these never talked to `Molecule.assign_partial_charges`/`Molecule.assign_fractional_bond_orders` at all, in addition to the existing note about topological representations vs. force field settings

Reply:
Mmm good smithing. Done!